client/star.svg

```diff
@@ -0,0 +1,3 @@
+<svg class="star" xmlns="http://www.w3.org/2000/svg" viewBox="3 3 18 18" aria-hidden="true" focusable="false">
```

Comment:
Is this how you're generating your star rating?  When we have some time, you'll have to show me how this works.  I was really curious about it as I was reading your review header component.